docs/getting_started/install_using_ddev.md

```diff
@@ -0,0 +1,384 @@
+
+For example, it can be done with `sed`:
+
+```bash
+sed -i 's/%IP_ADDRESS%/*/' .ddev/apache/apache-site.conf
+sed -i 's/%PORT%/80/' .ddev/apache/apache-site.conf
+sed -i 's/%BASEDIR%/\/var\/www\/html/' .ddev/apache/apache-site.conf
+sed -i 's/%BODY_SIZE_LIMIT%/0/' .ddev/apache/apache-site.conf
+sed -i 's/%TIMEOUT%/0/' .ddev/apache/apache-site.conf
+sed -i 's/%FASTCGI_PASS%/unix:\/var\/run\/php-fpm.sock/' .ddev/apache/apache-site.conf
+```
+
+Finally, restart the project:
+
+```bash
+ddev restart
+```
+
+##### Scripted
+
+Generate the Virtual Host using https://github.com/ibexa/docker/blob/main/scripts/vhost.sh:
+
+```bash
+curl -O https://raw.githubusercontent.com/ibexa/docker/main/scripts/vhost.sh
+bash vhost.sh --template-file=vendor/ibexa/post-install/resources/templates/apache2/vhost.template \
+  --ip='*' \
+  --host-name='my-ddev-project.ddev.site' \
+  --host-alias='*.ddev.site' \
+  --basedir='/var/www/html' \
+  --sf-env=dev \
+  > .ddev/apache/apache-site.conf
+rm vhost.sh
+```
+
+Then, restart the project:
+
+```bash
+ddev restart
+```
+
+### Run an already existing project
+
+- Instead of creating an empty directory like in _[1. Create a DDEV project directory](#1-create-a-ddev-project-directory)_, a directory already containing an Ibexa DXP project is used.
+- In _[2. Configure DDEV / Configure PHP version and document root](#configure-php-version-and-document-root)_, no need to create the Document root, `--create-docroot` mustn't be used.
+- Instead of `ddev composer create` from _[5. Create Ibexa DXP project](#5-create-ibexa-dxp-project)_, use solely `ddev composer install`.
+- [Ibexa data migration](../content_management/data_migration/importing_data.md) or [`ddev import-db`](https://ddev.readthedocs.io/en/latest/users/usage/commands/#import-db) can be used to populate the database.
+- Content's binary files can be copied into `public/var`.
+
+The following example run on DDEV a [version controlled project](https://doc.ibexa.co/en/latest/getting_started/install_ibexa_dxp/#add-project-to-version-control), a local clone of a remote Git repository:
+```bash
+git clone <repository> my-ddev-project && cd my-ddev-project
+.ddev/ >> .gitignore
+ddev config --project-type=php --php-version 8.1 \
+  --docroot=public \
+  --web-environment-add DATABASE_URL=mysql://db:db@db:3306/db \
+  --http-port=8080 --https-port=8443
+ddev start
+ddev composer config --global http-basic.updates.ibexa.co <installation-key> <token-password>
+ddev php bin/console ibexa:migrations:migrate --file=project_content_types.yaml
+ddev composer install
+ddev launch
+```
+
+Notice that the example choose to `.gitignore` the whole `.ddev/` directory. It can go otherwise. Some DDEV configs can be shared among developers. For example, a common `.ddev/config.yaml` can be committed for everyone and [locally extended or override](https://ddev.readthedocs.io/en/latest/users/extend/customization-extendibility/#extending-configyaml-with-custom-configyaml-files). 
```

Comment:
Done in 7b9728810712783a4b0216f0199de4808f9c93da

I changed "you can also version it" with "you can also version it" because there might be some unwanted things in it, like the `.ddev/homeadditions/.composer/auth.json`.